app.json

```diff
@@ -9,6 +9,7 @@
   },
   "addons": [
     "fastly:fast",
-    "logentries:starter"
+    "logentries:starter",
+    "redistogo"
```

Comment:
Heroku Redis is now generally available (`heroku-redis`). I would favour using it over a third party add-on.